dlio_benchmark/data_loader/dali_data_loader.py

```diff
@@ -140,19 +136,25 @@ def next(self):
         self.read(True)
         while step < self.num_samples // self.batch_size:
             for pipe in self.pipelines:
+                dft_ai.dataloader.fetch.start()
                 try:
                     outputs = pipe.share_outputs()
                 except StopIteration:
+                    # it is fine to not stop `dft_ai.dataloader.fetch` here since
+                    # it will be reset at the next run
                     return
+                dft_ai.dataloader.fetch.stop()
                 self.logger.debug(f"{utcnow()} Output batch {step} {len(outputs)}")
                 yield outputs
                 step += 1
-                dlp.update(step = step)
+                dlp.update(step=step)
```

Comment:
Why do we need dlp?

Reply:
we still use `dlp` in several places within that file.

I am happy to remove the `update` if it is not needed

Reply:
this is removed